catroid/src/main/java/org/catrobat/catroid/cloudmessaging/CloudMessage.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.cloudmessaging;
+
+import android.support.annotation.VisibleForTesting;
+
+import com.google.firebase.messaging.RemoteMessage;
+
+import java.util.HashMap;
+
+public class CloudMessage {
+
+	public static final String TITLE = "title";
+	public static final String MESSAGE = "message";
+	public static final String WEB_PAGE_URL = "link";
+
+	private String title;
+	private String message;
+	private String url;
+
+	@VisibleForTesting
+	public CloudMessage() {
+
+	}
+
+	@VisibleForTesting
+	public CloudMessage(HashMap<String, String> data) {
```

Comment:
Required [here](https://github.com/nishantt95/Catroid/blob/20a3eaab1f048d71e6afb6d5222978d9a8b0e6a1/catroid/src/androidTest/java/org/catrobat/catroid/test/cloudmessaging/CustomFirebaseMessagingServiceTest.java#L71-L74) or else I need to set the data one by one. This will help when the number of parameters will grow in future.